packages/datadog-shimmer/src/shimmer.js

```diff
@@ -131,7 +126,7 @@ function safeWrapper (original, wrapper) {
 
   // We're going to hold on to current callState in this variable in this scope,
   // which is fine because any time we reference it, we're referencing it synchronously.
-  // We'll use it in the our wrapper (which, again, is called syncrhonously), and in the
+  // We'll use it in the our wrapper (which, again, is called synchronously), and in the
```

Comment:
```suggestion
  // We'll use it in our wrapper (which, again, is called synchronously), and in the
```

Reply:
Obsolete due to the removal of safe mode :)